### DISCUSSION TITLE
How do you want to organize a lot of apps?

### DISCUSSION THREAD

**Discussion by ETWang1991** [IceWhaleTech/CasaOS#828] [Ideas]
If you have more than 10 APPs, how do you want to organize them? 5 options are proposed below, please click "UP" for the option you support.

**Comment by ETWang1991** [comment]
![IMG_8089](https://user-images.githubusercontent.com/109786132/213100049-596d7e07-c4ad-4cd3-81f5-22ece0bfccad.JPG)

**Reply by CosmoFox** [reply]
BEst!!!

**Reply by q2316367743** [reply]
这个感觉最好

**Reply by NP-compete** [reply]
I feel like this design would still cause an overflow, when the number of app increases.

**Reply by MustardtheCalico** [reply]
> I feel like this design would still cause an overflow, when the number of app increases.

I can see that being an issue, but if the pop up had a slider (like in option C) that would be the best combination of all the options - Or maybe a way to group apps together, and then decide whether the groups appear as option B/C, or as pop ups....

**Reply by Mahoo12138** [reply]
The size of My Favorite(multi-icons) is different from common's size, it looks incongruous.

**Reply by RubyEyedReaper** [reply]
this is how it needs to be done, simple yet effective, it would be nice to put all my arr programs in one folder

**Comment by ETWang1991** [comment]
![IMG_8090](https://user-images.githubusercontent.com/109786132/213100101-dd8924fd-8f94-49c0-80b9-095a108ff6dc.JPG)

**Reply by nghialele** [reply]
This one looking good!

**Comment by ETWang1991** [comment]
![IMG_8091](https://user-images.githubusercontent.com/109786132/213100146-ba725dfe-4992-4c6e-b9ba-35e23e057549.JPG)

**Reply by MustardtheCalico** [reply]
I prefer option A, but if there was the option to slide between pages within the pop up in the same way as discussed here, it might give the best of both world...

**Comment by ETWang1991** [comment]
![IMG_8092](https://user-images.githubusercontent.com/109786132/213100171-21793e1e-1b36-47a3-9290-4e9d5cfe9a09.JPG)

**Comment by ETWang1991** [comment]
![IMG_8093](https://user-images.githubusercontent.com/109786132/213100192-ee655c97-66ce-4ce1-8389-12c0d4d10e40.JPG)

**Comment by leesonaa** [comment]
什么时候支持这样设置呀？

**Comment by LinkLeong** [comment]
We are ready to implement this feature, here is our design, do you have a better suggestion?https://www.figma.com/file/ZNgzSUYKKRfc7DN86t1ses/V0.4.9?type=design&node-id=52%3A1114&mode=design&t=3i7wurt1oWR5U2Er-1

**Reply by elecompte** [reply]
Looks like you may have linked the wrong board. Interested to see which choice you decided on.

**Comment by Frozeph** [comment]
Can we bump this please, and why not include all options as a choice through settings
